src/Utils/systems.jl

```diff
@@ -95,27 +106,42 @@ julia> dim(add_dimension(X))
 julia> typeof(X)
 LazySets.ZeroSet{Float64}
 ```
+
+More than one dimension can be added passing the second argument:
+
+```jldoctest add_dimension_set
+julia> Xext = add_dimension(BallInf(zeros(10), 0.1), 4);
+
+julia> dim(Xext)
+14
+```
+
+### Notes
+
+In the special case that the given set is a zero set, instead of cartesian product
+a new zero set with extended dimensions is returned.
 """
-function add_dimension(X::LazySet)::LazySet
+function add_dimension(X::LazySet, m=1)::LazySet
     if X isa ZeroSet
-        return ZeroSet(dim(X)+1)
+        return ZeroSet(dim(X)+m)
     else
-        return X * ZeroSet(1)
+        return X * ZeroSet(m)
     end
 end
 
 """
-    add_dimension(cont_sys)
+    add_dimension(cont_sys, m=1)
```

Comment:
If you want to modify, you could rename `cont_sys` to `cs` as in the source code.